### DISCUSSION TITLE
Does the adaptive mode for 60hz or 120hz use power every time it switches? Because if so I don't think it's worth it on non adaptive devices.

### DISCUSSION THREAD

**Discussion by Xisrr1** [tribalfs/GalaxyMaxHzPub#164] [Q&A] [answered]

**Comment by tribalfs** [comment, marked as answer]
I don't think it's that much, only very minimal.  It's best to set the optimal switching down interval according to your usage. The more frequent you interact with the screen set the delay a little bit longer.
The biggest issue on non-adaptive is the gamma consistency.

**Reply by Xisrr1** [reply]
Gamma?

**Reply by tribalfs** [reply]
Check related info in [FAQ # 8](https://github.com/tribalfs/GalaxyMaxHzPub/wiki/Frequently-Asked-Questions-(FAQs)#8-why-sometimes-i-notice-my-color-shift-when-refresh-rate-switches-in-low-brightness-whats-the-workaround)

**Reply by Xisrr1** [reply]
Oh yes, it doesn't bother me usually. Unless I am in a dark place.
